float64 agv_delta_voltage          # lượng giảm điện áp
float64 agv_time_decrease_voltage  # Thời gian giảm điện áp
int16 agv_waitStop               # lỗi dừng chờ 
float64 agv_time_waitStop          # thời gian dừng chờ
int16 agv_common_error           # chạm badersoc, ko phát hiện gương, mất kết nối với mạch   >> xử lý lỗi thì AGV chạy tiếp

# int16 agv2_voltage                
# int16 agv2_delta_voltage          
# int16 agv2_time_decrease_voltage  
# int16 agv2_waitStop                
# int16 agv2_time_waitStop          
# int16 agv2_common_error           

# int16 agv3_voltage                
# int16 agv3_delta_voltage          
# int16 agv3_time_decrease_voltage  
# int16 agv3_waitStop                
# int16 agv3_time_waitStop          
# int16 agv3_common_error          